public_html/layouts/basic/modules/Settings/WebserviceApps/resources/WebserviceApps.js

```diff
@@ -87,14 +120,35 @@ jQuery.Class('Settings_WebserviceApps_Index_Js', {}, {
 			});
 		});
 	},
+	/**
+	 * Register buttons to copy api key
+	 */
+	registerCopyApiKey: function () {
+		new Clipboard('.copyApiKey', {
```

Comment:
This password functionality should be universal and added in a file https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/public_html/layouts/resources/Fields.js